tests/databases/test_pymongo.py

```diff
@@ -140,7 +142,7 @@ def test_database_total_score_eleven(data_single_filter_logical_company_code, da
     assert total_score == 11
 
 
-'''
+@pytest.mark.skip(reason="Cannot mock random sampling")
```

Comment:
Not sure why this is skipped. If the test is useless, perhaps delete it and do this in another PR. Seems not relevant to the topic of this bugfix PR.

Reply:
That is why I split changes into multiple commits -- to not lose track of the changes. For instance, I still need to update docs. Every single improvement to the existing codebase should be welcomed.

Reply:
it is true and it is welcomed. But it is no problem to have smaller, topic-focused PRs. And update  relevant docs in another PR when the bug is fixed even tho for some time there would be no mention of new configuration in documentation after merge. Issue with label documentation is enough and then have review of different PR.

Reply:
You are totally right, I opened the new PR https://github.com/SAP/odfuzz/pull/49 for this.